src/raspbian/10/helix/arm32v6/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM tianon/raspbian:buster
+# There doesn't seem to be an official Raspberry Pi OS Docker image
+# This is s compromise - it's a repo run by Tianon Gravi, the engineer
+# responsible for the Docker Hub official images repo - this is as close
+# to official as it's possible to be, without being official.
```

Comment:
I'd like to have sign-off from @blowdart before we take this.

Reply:
cc @mmitche as well

Reply:
IMO we shouldn't pull directly from another user's image, at least not without a specific, immutable version reference.

Reply:
This. And we shouldn't do it at unless we're going to officially build for raspberry pis. If we're not, then it gets left to the community.

Reply:
We still need most of the runtime changes (e.g. additions to Interop.ProcessorArchitecture.cs) whether we plan to leave the actual final runtime pack publication to the community or not, and if we want to validate changes, we either need a Docker image we can run on Raspbian.9.Arm32.IoT.Open, or we need a pool with Raspbian 10 (our support matrix is for Debian 10+)

I'm fine with duplicating the scripts from debuerreotype for base image creation (i.e. `FROM scratch`) to remove the dependency on another user repo, although obviously that significantly increases the complexity of the image creation for any future maintenance. It's actually very close to the logic in build-rootfs.sh

....

_could_ I just use build-rootfs.sh? Investigating...

Reply:
The answer is "yes"

It's a little slower, and a little bigger, but we absolutely can generate a Docker image based off a crossrootfs tarball, meaning no third party docker images needed at all for any arch we can generate a cross for.

Reply:
I'm hitting resolve on this specific issue since it's taken care of, fine with discussion continuing in general though